code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -633,19 +584,17 @@ generate/load female uniform sprites matching all previously decided variables
 					break
 
 // Only renders the head of the human
-/mob/living/carbon/human/proc/update_body_parts_head_only()
-	if (!dna)
-		return
-
-	if (!dna.species)
+// Only renders the head of the human
+/mob/living/carbon/human/proc/update_body_parts_head_only(var/update_limb_data)
```

Comment:
remove var/ from arguments, not necessary

Reply:
actually is one the reason why the pr's failing linters

Reply:
its also failing linters because this pr is half finished in a literal sense.